plugins/delphes/examples/EDM4Hep2LCIO_example.py

```diff
@@ -45,22 +54,32 @@ def generateEvents(inputFileName, outputFileName, nEvents):
         tracks = IMPL.LCCollectionVec(EVENT.LCIO.TRACK)
         recops = IMPL.LCCollectionVec(EVENT.LCIO.RECONSTRUCTEDPARTICLE)
 
+        #print ('nreco part  ',e.ReconstructedParticles.size(),'  ntracks  ',e.EFlowTrack.size())
+
         #loop over the reconstructed particles
         for rp in range(e.ReconstructedParticles.size()):
```

Comment:
I am not entirely sure, but can't we do something like
```python
for rp in e.ReconstructedParticles:
  # do something
```

If not we should definitely add that to our TODO list

Reply:
yes we could in principle, I thought the index was needed for some earlier checks, I will anyway improve this script later on, thanks for reminding it to me